content/articles/standard-vs-letsencrypt.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: Standard vs Let's Encrypt SSL Certificates
+excerpt: This article summarizes the most important DNSimple offering differences between Let's Encrypt and Standard SSL certificates.
+categories:
+- SSL Certificates
+---
+
+# Standard vs Let's Encrypt SSL Certificates
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+<callout>
+In this article we'll use the term _standard_ as described in our [TLS/SSL certificate page](/articles/ssl-certificates#standard-certificate).
+</callout>
+
+## Comparison Let's Encrypt vs Standard SSL certificates
+
+The table below summarizes the most important DNSimple offering differences between Let's Encrypt and Standard SSL certificates. These differences may help you to decide which certificate you need.
+
+<note>
+Please note that the table only reflects the status of the current DNSimple offering. Some features may be available at the Certificate Authority, but currently not supported by DNSimple.
+</note>
+
+|               | Let's Encrypt | Standard      |
+|---------------+---------------+---------------|
+Certificate Expiration | 90 days | 1-3 years
+Single names | Supported | Supported
+Custom Single names | Supported (on certain plans) | Supported
+Wildcard names | Not Supported (by CA) | Supported
+Same-domain multi-names (SAN) | Supported (on certain plans) | Not Supported

```

Comment:
Do we want to be more specific about what support on each plan by mentioning the plan?